spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -132,9 +152,12 @@ The value of the gauge must be equivalent to a call to the instance ConcurrentGa
 [source, json]
 ----
 {
-  "callCount": 48
-  "callCount_min": 4
-  "callCount_max": 50
+  "callCount": 48,
```

Comment:
Shouldn't the concurrent gauge, because it is a compound metric, follow the same pattern as other compound metrics (meter, histogram, timer)?

Reply:
I suppose it should. I was modifying what was already there.
Should the current count be labeled as `current` or something else?

I remember there was a discussion on using `_current` to suffix the current value of the CG, but it was ruled out.

```
{
"callCount" {
       "<something>" : 1,
       "min" : 1,
       "max" : 1
 }
}

Reply:
I'd say `current` makes the most sense

Reply:
Alright, latest commit is `current` unless there are further objections ;)